.env.example

```diff
@@ -8,4 +8,5 @@ POSTGRES_URI_BOT=sqlite:///
 # Variables for various command cogs.
 AOC_SESSION_ID=
 SB_BASE_THRESHOLD=8
-SB_BIG_THRESHOLD=24
\ No newline at end of file
+SB_BIG_THRESHOLD=24
+SB_RATELIMIT=30
```

Comment:
https://github.com/UQComputingSociety/uqcsbot-discord/wiki/Tokens-and-Environment-Variables#sb_base_threshold-sb_big_threshold-sb_ratelimit

...this page should probably be better documented.

Reply:
I did not see that page. Maybe add a link to it within the code?

Reply:
It'd be better to link it more prominently in the README, because it serves as documentation for all our env variables (and in general, should also have more detailed instructions on everything else that the bot needs)